docs/proposals/apb-remove-image-field.md

```diff
@@ -0,0 +1,75 @@
+# Remove Image Field from APB Spec Proposal
+
+## Introduction
+
+The broker currently relies on the image field of the
+[apb.yaml](https://github.com/fusor/ansible-playbook-bundle/blob/master/src/apb/dat/apb.yml.j2)
+to determine the location of the docker image to deploy instead of relying on the
+registry adapters to formulate the image path based on the location of the spec
```

Comment:
How will this handle having multiple adaptors with duplicate images?

Reply:
I'm not certain that I follow you. I feel like what you may be suggesting is the case where the broker is configured like so (using @dymurray since he actually has apb's in his org):

```
registry:
  - type: dockerhub
    name: dh
    url: https://registry.hub.docker.com
    user: changeme
    pass: changeme
    org: ansibleplaybookbundle
  - type: dockerhub
    name: dy
    url: https://registry.hub.docker.com
    user: changeme
    pass: changeme
    org: dymurray
```

And the images are stored in etcd, looking at MediaWiki since there is one in each.

For ansibleplaybookbundle organization:
```
{ ... "image":"ansibleplaybookbundle/mediawiki123-apb:latest" ...}
```

For dymurray organization:
```
{... "image":"dymurray/mediawiki123-apb:latest" ...}
```

Is this what you are referring to?

Reply:
`FQName` was introduced to uniquely identify APBs across adapters; it can be considered our "primary key", unless I'm misunderstanding this.